src/components/buttons/ErrorActionButton.vue

```diff
@@ -0,0 +1,28 @@
+<template>

```

Comment:
To not undo ebkr's progress, any new components that we add should IMO use the composition API.